app/src/main/res/values/strings.xml

```diff
@@ -1764,5 +1764,6 @@
 
     <!-- Content description for privacy content close button -->
     <string name="privacy_content_close_button_content_description">Close</string>
+    <string name="mozac_service_nimbus_use_preview_collection">Use Nimbus Preview Collection (requires restart)</string>
```

Comment:
Can you move this to `static_strings.xml`? We typically keep our secret settings menu strings there since we don't expect that it needs to be localized since it's internal only. Another note is that we typically reserve the "mozac" prefix for strings/xml coming from AC. I think we will want to follow the other preference naming convention here. See "preferences_debug_settings_enable_address_feature" for an example for both the static_strings and preference naming.

Reply:
> Can you move this to `static_strings.xml`? We typically keep our secret settings menu strings there since we don't expect that it needs to be localized since it's internal only. Another note is that we typically reserve the "mozac" prefix for strings/xml coming from AC. I think we will want to follow the other preference naming convention here. See "preferences_debug_settings_enable_address_feature" for an example for both the static_strings and preference naming.

Sure thing!  Thanks for the info on the naming conventions.